documentation/modules/exploit/multi/http/pimcore_unserialize_rce.md

```diff
@@ -0,0 +1,109 @@
+## Description
+
+This module exploits a PHP unserialize() in Pimcore before 5.7.1 to execute arbitrary code. An authenticated user with "classes" permission could exploit the vulnerability.
+
+The vulnerability exists in the "ClassController.php" class, where the "bulk-commit" method make it possible to exploit the unserialize function when passing untrusted values in "data" parameter.
+
+Tested on Pimcore 5.6.6, 5.6.5, 5.6.4, 5.6.3, 5.6.2, 5.6.1, 5.6.0, 5.5.4, 5.5.3, 5.5.2, 5.5.1, 5.4.4, 5.4.3, 5.4.2, 5.4.1, 5.4.0 with the Symfony unserialize payload
+
+Tested on Pimcore 4.6.5, 4.6.4, 4.6.3, 4.6.2, 4.6.1, 4.6.0, 4.5.0, 4.4.3, 4.4.2, 4.4.1, 4.4.0, 4.3.1, 4.3.0, 4.2.0, 4.1.3, 4.1.2, 4.1.1, 4.1.0, 4.0.1, 4.0.0 with the Zend unserialize payload
+
+## Vulnerable Application
+
+Affecting Pimcore, version 5.x <= 5.6.6 and 4.x
```

Comment:
this is covered above, and can be removed

Reply:
Hi @h00die, from an analysis of the source code, all the brach 4.x and 5.x before 5.6.6 is vulnerable but for reasons of time I have tested only the versions indicated before so "tested" and "affected" are different.

With a little more time I can try the missing versions.

Reply:
You've tested a lot of versions already, no *need* to test additional.  Was just trying to save a little bit of text